### ISSUE TITLE
lightweight network for self-teaching

### ISSUE THREAD

**Issue by lnSong** [mattpoggi/mono-uncertainty#27]
When I use monodepth2 and implement self-teaching, uncertainty does improve the results. But why I use a lightweight network and implement self-teaching, the result is not better but worse?

**Comment by mattpoggi** [comment]
Can you give more details about the lightweight model and the training protocol (epochs, etc.)?

**Comment by lnSong** [comment]
I use mobilenet as the backbone, the epoch is 100, and the learning rate is 0.0001. I first train the teacher network, then fix the teacher network, and then use mobilenet with uncertainty prediction branch as the student network. The teacher network is the same as the student network.Thank you!
